app/Models/Category.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    use HasFactory;
+
+    protected $guarded = [];
+
+    public function path()
```

Comment:
Why you doing it like this? I see no point at all. Plus in blade files in some places you use `route()` in some your own method `path()` no consistency. Just do for example `route('categories.edit', $category)`